<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>defectvalue</class>
 <widget class="QWidget" name="defectvalue">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>226</width>
    <height>295</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Width</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Score</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Height</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QDoubleSpinBox" name="doubleSpinBox_Width">
     <property name="decimals">
      <number>6</number>
     </property>
     <property name="maximum">
      <double>999.990000000000009</double>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QDoubleSpinBox" name="doubleSpinBox_Score">
     <property name="decimals">
      <number>6</number>
     </property>
     <property name="maximum">
      <double>999.990000000000009</double>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QDoubleSpinBox" name="doubleSpinBox_Contrast">
     <property name="decimals">
      <number>6</number>
     </property>
     <property name="maximum">
      <double>999.990000000000009</double>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QDoubleSpinBox" name="doubleSpinBox_SkeletonLength">
     <property name="decimals">
      <number>6</number>
     </property>
     <property name="maximum">
      <double>999.990000000000009</double>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QDoubleSpinBox" name="doubleSpinBox_Area">
     <property name="decimals">
      <number>6</number>
     </property>
     <property name="maximum">
      <double>999.990000000000009</double>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>SkeletonLength</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QDoubleSpinBox" name="doubleSpinBox_Height">
     <property name="decimals">
      <number>6</number>
     </property>
     <property name="maximum">
      <double>999.990000000000009</double>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Area</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Contrast</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>PosX</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QDoubleSpinBox" name="doubleSpinBox_PosX">
     <property name="decimals">
      <number>6</number>
     </property>
     <property name="maximum">
      <double>999.990000000000009</double>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string>PosY</string>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <widget class="QDoubleSpinBox" name="doubleSpinBox_PosY">
     <property name="decimals">
      <number>6</number>
     </property>
    </widget>
   </item>
   <item row="11" column="1">
    <widget class="QPushButton" name="pushButton_cancel">
     <property name="text">
      <string>cancel</string>
     </property>
    </widget>
   </item>
   <item row="11" column="0">
    <widget class="QPushButton" name="pushButton_ok">
     <property name="text">
      <string>OK</string>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <widget class="QLabel" name="label_9">
     <property name="text">
      <string>Svm</string>
     </property>
    </widget>
   </item>
   <item row="9" column="1">
    <widget class="QSpinBox" name="spinBox_Svm">
     <property name="minimum">
      <number>-1</number>
     </property>
     <property name="maximum">
      <number>2</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
